### DISCUSSION TITLE
--get-id

### DISCUSSION THREAD

**Discussion by 593172440** [rustdesk/rustdesk#4538] [Q&A]
Ask how to use this function??

[rustdesk/src/core_main.rs](https://github.com/rustdesk/rustdesk/blob/789200853575f63a336bce72c1a09a215ecfb5cb/src/core_main.rs#L223)

Line 223 in [7892008](https://github.com/rustdesk/rustdesk/commit/789200853575f63a336bce72c1a09a215ecfb5cb)

 } else if args[0] == "--get-id" {

**Comment by SergeyMy** [comment]
`sudo ./rustdesk --get-id`

**Reply by ghost** [reply]
Works on macOS but also launches the RD software.

Does nothing on Windows.

**Reply by dinger1986** [reply]
works fine from system
